tests/foreman/cli/test_repository.py

```diff
@@ -908,6 +908,40 @@ def test_positive_synchronize_docker_repo(self, repo, module_product, module_org
             ]
         )
 
+    @pytest.mark.tier2
+    @pytest.mark.parametrize(
+        'repo_options',
+        **parametrized(
+            [
+                {
+                    'content-type': 'docker',
+                    'docker-upstream-name': CONTAINER_UPSTREAM_NAME,
+                    'name': valid_docker_repository_names()[0],
+                    'url': CONTAINER_REGISTRY_HUB,
+                }
+            ]
+        ),
+        indirect=True,
+    )
+    def test_verify_checksum_non_yum_repo(self, repo):
```

Comment:
Mentioned BZ says `Verify Content Checksum, on container repositories`, and as test verifies only docker repository
```suggestion
    def test_verify_checksum_container_repo(self, repo):
```